unstructured_inference/models/base.py

```diff
@@ -23,7 +23,7 @@
     UnstructuredYoloXModel,
 )
 
-DEFAULT_MODEL = "detectron2_onnx"
+DEFAULT_MODEL = "yolox_quantized"
```

Comment:
maybe we should use a config pattern that allows this to be set via env variable?

Reply:
@cragwolfe, what do you think? 
I can get this string from an environment variable, if isn't present I can keep the model defined in this file.

Reply:
a change of config pattern is better a separate PR in the first place, don't need to mix that into this PR